dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1655,6 +1655,7 @@ FROM
     ('ethx-stader-ethx','ethereum','ETHx',0xA35b1B31Ce002FBF2058D22F30f95D405200A15b, 18),
     ('rseth-rseth','ethereum','rsETH',0xa1290d69c65a6fe4df752f95823fae25cb99e5a7, 18),
     ('pufeth-pufeth','ethereum','pufETH',0xD9A442856C234a39a81a089C06451EBAa4306a72, 18),
+    ('puffer-puffer-finance','ethereum','PUFFER',0x4d1c297d39c5c1277964d0e3f8aa901493664530, 18),
```

Comment:
```suggestion
    ('puffer-puffer','ethereum','PUFFER',0x4d1c297d39c5c1277964d0e3f8aa901493664530, 18),
```

Reply:
plz validate on coinpaprika, where we read from.
https://coinpaprika.com/coin/puffer-puffer/
can find id in URL or the API id on page